template/src/home/HomePage.tsx

```diff
@@ -6,15 +8,37 @@ function HomePage() {
     <Page title={"Home"}>
       <PageContent>
         <h1>{"Home"}</h1>
-        
+
         <p>{"This is the home page of the application."}</p>
         <p>
           <b>{"Active environment: "}</b>
           {process.env.REACT_APP_BUILD_ENVIRONMENT || "Not defined"}
         </p>
+
+        <br />
+
+        <h3>{"Hipo - React UI Toolkit Component Examples"}</h3>
+
+        <div>
+          <HipoButton type={"button"} onClick={onClick}>
+            {"Click Me"}
+          </HipoButton>
+
+          <HipoButton type={"button"} onClick={onClick} isDisabled={true}>
+            {"Click Me - isDisabled"}
+          </HipoButton>
+
+          <HipoButton type={"button"} shouldDisplaySpinner={true}>
+            {"With Spinner"}
+          </HipoButton>
+        </div>
```

Comment:
We can add a components page like you have in Otherverse. We can turn that into an across-project practice. We can add a link for that page in the PageHeader and make it so that it only appears when in the development mode.

Reply:
Good idea 👍 @yigiterdev If you have time, I think we can do this in this PR, if not, let's create an issue for this.